### PR TITLE
Corrections added

### PR DESCRIPTION
**Pull request by jbnetcat**
No comments really.

### CODE REVIEW

File: knowledge/jonathan/Bash-Guide-WCSC/_config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (SilexOne) [inline knowledge/jonathan/Bash-Guide-WCSC/_config.yml]: Get rid of this file.

File: knowledge/jonathan/Bash-Guide-WCSC/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# By Jonathan B
+
+The purpose of this **Bash** guide/tutorial is to assist new and seasoned members of the **WhiteHatters Computer Security Club**. It also serves as a handy online quick-reference for tips, tricks, and commands relevant to computer security professionals. We're all in this to learn together, any suggestions for this guide are greatly appreciated.
```
Comment (SilexOne) [inline knowledge/jonathan/Bash-Guide-WCSC/README.md]: Move the content of this file on top of your bash guide then delete this file.

**Review by SilexOne — CHANGES_REQUESTED** [review]
I would prefer markdown but HTML is fine. Review the inline code comments.

**Comment by jbnetcat** [comment]
Made 3 changes

**Comment by jbnetcat** [comment]
Updates applied.

**Comment by jbnetcat** [comment]
Changes applied

**Review by SilexOne — APPROVED** [review]
cool story!

**Pull request merged**